v1.23/asuscloudinfra/PRODUCT.yaml

```diff
@@ -0,0 +1,8 @@
+vendor: ASUS Cloud
+name: ASUS Cloud Infra 
+version: 4.0.0
+website_url: www.asuscloud.com
```

Comment:
can you please add '[http://](http:)' or '[https://](https://github.com/cncf/k8s-conformance/pull/1849/files)'to the URL to help the Bot to check it.